phases/00-setup-and-tooling/08-editor-setup/docs/en.md

```
@@ -122,7 +122,7 @@ Useful shortcuts:
 
 | Action | macOS | Linux/Windows |
 |--------|-------|---------------|
-| Toggle terminal | `` Ctrl+` `` | `` Ctrl+` `` |
+| Toggle terminal | `` Ctrl+` `` \| `` Ctrl+` `` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 2; Too few cells, row will be missing data

(MD056, table-column-count)